fe/fe-core/src/main/java/org/apache/doris/nereids/processor/post/RuntimeFilterGenerator.java

```diff
@@ -94,122 +113,30 @@ public PhysicalPlan visitPhysicalHashJoin(PhysicalHashJoin<? extends Plan, ? ext
         Map<NamedExpression, Pair<PhysicalRelation, Slot>> aliasTransferMap = ctx.getAliasTransferMap();
         join.right().accept(this, context);
         join.left().accept(this, context);
+        collectPushDownCTEInfos(join, context);
```

Comment:
i think we should not collect push down cte info, when current join node satisfy `DENIED_JOIN_TYPES.contains(join.getJoinType()) || join.isMarkJoin()`

Reply:
done